package.json

```diff
@@ -25,11 +25,13 @@
   "homepage": "https://github.com/Ryhthmic-Squad/grace-shopper#readme",
   "dependencies": {
     "bcrypt": "^5.0.1",
+    "connect": "^3.7.0",
```

Comment:
I don't think we need this package&mdash;looks like [connect](https://www.npmjs.com/package/connect) is a server-side wrapper around http, like express.

Reply:
Ok, thanks I installed connect when I meant react-redux